src/sunburst.js

```diff
@@ -51,7 +50,8 @@ export default Kapsule({
       }
     },
     onClick: { triggerUpdate: false },
-    onHover: { triggerUpdate: false }
+    onHover: { triggerUpdate: false },
+    animationDuration: { default: 750 }
```

Comment:
Changing this value doesn't require an update, since it will be picked up the next time there is a transition. Thus, you can set `triggerUpdate: false`. Also, let's name the attribute `transitionDuration` instead as that's more descriptive of its intent.